node-repository/src/main/java/com/yahoo/vespa/hosted/provision/Node.java

```diff
@@ -476,7 +476,10 @@ public String toString() {
         parked,
 
         /** This host has previously been in use but is now removed. */
-        deprovisioned;
+        deprovisioned,
+
+        /** This host is currently undergoing repair. */
+        breakfixed;
```

Comment:
Why do we need a new state?

Valid state transitions to/from this should be documented in `NodeRepository` (at the top).

Reply:
Fixed